### PR TITLE
Moved from a single Dockerfile to a more convenient docker-compose setup

### PR DESCRIPTION
**Pull request by BenHUET**

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+version: "3"
+
+services:
+  mysql:
+    image: mysql:latest
```
Comment (leonjza) [inline docker/docker-compose.yml]: Could you fix these line breaks please?

**Comment by BenHUET** [comment]
It's done, sorry about that...

**Comment by leonjza** [comment]
Sorry for taking so long with this. I'd like to keep the single docker container we have as well. Could we perhaps move the old files to a single docker container folder, and then your `docker-compose` setup to a compose folder?

Thanks again!

**Comment by BenHUET** [comment]
its done !

**Comment by leonjza** [comment]
Awesome. Ill probably merge this today!

**Comment by leonjza** [comment]
Finally :D

**Pull request merged**